### PR TITLE
Add search function to list pages

### PR DESCRIPTION
**Pull request by Rubikium**
refs #10

### CODE REVIEW

File: django_material_demo/django_material_demo/settings.py [hunk 1]
```diff
@@ -81,6 +82,28 @@
...
+        }
+    },
+    'loggers': {
+        'django.db.backends': {
+            'level': 'DEBUG',
+            'handlers': ['console'],
+        }
+    }
+}
+
```
Comment (Steven-Chan) [inline django_material_demo/django_material_demo/settings.py:106]: Pls help put this to a separate commit.

**Review by Steven-Chan — CHANGES_REQUESTED** [review]
Overall looks good, just minor comment.

File: django_material_demo/polls/admin_views.py [hunk 2]
```diff
@@ -378,13 +413,36 @@ def get_form_class(self):
...
+class QuestionFilterForm(forms.Form):
+    layout = Layout('search',
+                    'question_text',
+                    'show_vote')
+
+
+class QuestionFilter(SearchFilterMixin, FilterSet):
+    search_fields = ['question_text', 'creator__account__username',
+                     'choice__choice_text']
+    search = CharFilter(method='keyword_search', label='Search')
```
Comment (tung2744) [inline django_material_demo/polls/admin_views.py:425]: Looks like this line
```
search = CharFilter(method='keyword_search', label='Search')
```
Should be contained in `SearchFilterMixin`, because having this line without the mixin will not work.

Comment (Rubikium) [reply django_material_demo/polls/admin_views.py:425]: I tried that before, but the `FilterSet` cannot recognize `search` as one of its filter when it is in the mixin. I will look further into this tomorrow
